code/mob/living/carbon/human.dm

```diff
@@ -1101,11 +1101,6 @@
 
 	I.set_loc(src.loc)
 
-	// u_equip() already calls I.dropped()
-	// no it doesn't
-	if (isitem(I))
-		I.dropped(src) // let it know it's been dropped
```

Comment:
It does, for humans, I changed it so that it applies to everyone else too. This code resulted in double `dropped()` calls